ansible/library/bgp_route.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/python
+
+DOCUMENTATION = '''
+module:  bgp_route
+version_added:  "2.0"
+short_description: Retrieve BGP routing information from Quagga
+description:
+    - Retrieve BGP routing information from Quagga, using the VTYSH command line
+    - module argument is the property of "SHOW IP BGP"
+    - Current supported parsing commands:
+        * show ip bgp 100.0.0.1  ### show ip bgp prefix info
+        * show ip bgp neighbor 10.0.0.1 adv   ### show ip bgp neighbor address advertised routes
+    - Retrieved facts will be inserted into the 'bgp_rt'  or 'bgp_rt_neiadv'
+'''
```

Comment:
still lacking proper documentation of the module, such as requirements, options sections.

Reply:
[http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html](http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html)

---
In reply to: [142325420](https://github.com/Azure/sonic-mgmt/pull/284#discussion_r142325420) [](ancestors = 142325420)